File: culturio-src/web/visits/models/visitDto.ts
export class VisitDto {
  public id: number;
  public userId: number;
  public cultureObjectId: number;
  public timeOfVisit: Date;
  constructor(
    id: number,
    userId: number,
    cultureObjectId: number,
    timeOfVisit: Date
  ) {
    this.id = id;
    this.userId = userId;
    this.cultureObjectId = cultureObjectId;
    this.timeOfVisit = timeOfVisit;
  }
}

File: culturio-src/application/Visits/Models/VisitDto.cs
using System;
namespace Culturio.Application.Visits.Models
{
    public class VisitDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int CultureObjectId { get; set; }
        public DateTime TimeOfVisit { get; set; }
    }
}

